src/main/java/Nehsik.java

```diff
@@ -0,0 +1,127 @@
+import java.util.Scanner;
+public class Nehsik {
+    public static final int MAX_TASKS = 100;
+    public static final int MARK_TASK_INDEX = 5;
+    public static final int UNMARK_TASK_INDEX = 7;
+    public static final int TODO_DESCRIPTION_POSITION = 5;
+
+    public static void main(String[] args) {
+        displayGreetings();
+        
+        Scanner in = new Scanner(System.in);
+        Task[] taskList = new Task[MAX_TASKS];
+        int currTaskCount = 0;
+
+        while (true) {
+            String command = in.nextLine().trim();
+            if (command.equals("list")) {
+                displayTaskList(currTaskCount, taskList);
+            } else if (command.startsWith("mark")){
+                markTask(command, taskList);
+            } else if (command.startsWith("unmark")) {
+                unmarkTask(command, taskList);
```

Comment:
```suggestion
            if (command.equals("list")) {
                displayTaskList(currTaskCount, taskList);
            } else if (command.startsWith("mark")) {
                markTask(command, taskList);
            } else if (command.startsWith("unmark")) {
                unmarkTask(command, taskList);
```
Perhaps you might want to add a space between the parenthesis and the curly brace to be consistent with other lines of code.